src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindOutputPresenter.java

```diff
@@ -497,6 +499,7 @@ public void findInFilesBeginFind()
 
       // find result always starts with !replaceMode
       view_.setReplaceMode(false);
+      view_.turnOffReplaceMode();
```

Comment:
This fixes a pre-existing bug in the Find in Files. Previously, if I did a find in files search, tabbed over to the replace mode, entered a replacement in the text box, and then did Cmd + Shift + F to start a new find in files search, a new find in files search would open with the replace toolbar visible but disabled, and the previously entered replacement visible in the text box but not actually showing in the preview. This led to a weird UX where the user would have to edit the replacement text box, even if they wanted to keep the replacement the same, in order to actually activate replace mode. 

The fix is to make sure, when we begin a new Find in Files search, we completely turn off replace mode, not just turn off select parts of it and allow other parts to persist. This way when a new search begins, the replacement toolbar will not be visible, but if the user toggles over to it, the replacement they previously entered in the replacement text box will still persist, in case they want to reuse it.

Reply:
Good idea -- I haven't written any yet, so I'll have to look into how to do that, but seems like a good idea.

Also, I believe `turnOffReplaceMode();` sets replace mode to false so `view_.setReplaceMode(false);` is now redundant/unnecessary -- will remove.

Reply:
Looks like we only have some very basic ones for Find in Files so far: https://github.com/rstudio/rstudio-ide-automation/blob/main/rstudio_server_pro/tests/Panes/test_FindInFilesPane.py
It would be great if we could add some more, let me know if you need help getting started.

Reply:
Thanks, even so that's a very helpful starting point! I'll work on adding to that file